test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -263,13 +263,16 @@ public void testUrlColumnModeServesResult() throws Exception {
       String content = "from a yellow url connection comes monty python";
       writeDataToFile(testFile, content);
       ResponseGenerator resgen = ResponseGenerator.urlColumn(cfg);
-      URL testUrl = testFile.toURI().toURL();
-      ResultSet rs = makeMockUrlResultSet(testUrl);
+      // File.toURI() produces file:/path/to/temp/file, which is invalid.
```

Comment:
So the prod code sets uar.displayUrl to new URI(testFile.toURI().toString()) and that matches your reconstructed URI? Are you intentionally using a different construction here in the test? What about testFile.toURI().toURL().toURI()?